pkg/sql/schema_resolver.go

```diff
@@ -150,10 +150,47 @@ func (sr *schemaResolver) LookupObject(
 		}
 	}
 
-	prefix, objMeta, err = sr.descCollection.GetObjectByName(ctx, sr.txn, dbName, scName, obName, flags)
+	prefix, objMeta, err = sr.GetObjectByName(
+		ctx, dbName, scName, obName, flags,
+	)
 	return objMeta != nil, prefix, objMeta, err
 }
 
+// GetObjectByName returns an object descriptor by name.
```

Comment:
I think this comment should say something more about the structure of the various arguments. What is allowed to be empty and what is not?